src/storage.ts

```diff
@@ -12,6 +12,20 @@ export const getLastBlockAlerted = async (): Promise<number | undefined> => {
   return parseInt(foundVal);
 };
 
+export const setAlertedTxn = async (
+  type: "discord" | "twitter",
+  txnHash: string
+) => {
+  await setRedisAsync(`alerted:${type}:${txnHash.toLowerCase()}`, "true");
```

Comment:
super nit but we could make something like:
```ts
const getAlertedCacheKey = (type: "discord" | "twitter", txHash: string): string =>
  `alerted:${type}:${txnHash.toLowerCase()}`;
```

and then this could be
```suggestion
  const cacheKey = getAlertedCacheKey(type, txnHash);
  await setRedisAsync(cacheKey, "true");
```